modules/integration_azure-container-instance/common-modules.tf

```diff
@@ -0,0 +1 @@
+../../common/module/modules.tf
```

Comment:
this file must be deleted, and you should use the Azure specific one (with azure tags)

Reply:
File replaced by the Azure one